app/forms/webauthn_setup_form.rb

```diff
@@ -92,6 +101,21 @@ def safe_response(original_origin)
     false
   end
 
+  def process_authenticator_data_value(data_value)
+    return unless data_value.present?
+    # Ensure that its an int
+    return unless data_value.scan(/\D/).empty?
+    data_int_value = data_value.to_i
```

Comment:
I'd consider something like:
```suggestion
    data_int_value = Integer(data_value, 10)
    # ... the rest here
    rescue ArgumentError
    end
```

Reply:
(because `Integer()` is very strict, it'll throw on letters, floating points, etc etc)